src/main/java/li/cil/oc2/common/block/ComputerBlock.java

```diff
@@ -127,8 +128,8 @@ public ActionResultType onBlockActivated(final BlockState state, final World wor
         final ComputerTileEntity computer = (ComputerTileEntity) tileEntity;
         final ItemStack heldItem = player.getHeldItem(hand);
         if (Wrenches.isWrench(heldItem)) {
-            if (player instanceof ServerPlayerEntity) {
-                openContainerScreen(computer, (ServerPlayerEntity) player);
+            if (!world.isRemote()) {
```

Comment:
Why this change? In particular removing the instanceof check can potentially lead to an unsafe cast not. I'd understand adding the `!world.isRemote()` check, but removing the type check seems potentially dangerous?

So I'd say either revert this, or just add the world check (instead of replacing the instanceof check). Though usually having a server player should be a good enough indicator of being on the server.

Reply:
i had some problems without .isRemote() i will add both .isRemote() and instanceof